2024/day22/solution.md

```
@@ -0,0 +1,116 @@
+     
+    if [ -d "DevOps-Installation" ]; then
+        echo "Directory exists."
+    
+        # List the contents of the repository
+        ls DevOps-Installation
+    else
+        echo "Cloning the repository..."
+        git clone https://github.com/Amitabh-DevOps/DevOps-Installation.git
+    
+        # List the contents of the repository
+	    ls DevOps-Installation
```

Markdownlint (0.37.0)

Column: 1
Hard tabs

(MD010, no-hard-tabs)